assets/frontend/components/BookingFlow/BookingFlowCalendar/BookingFlowCalendar.tsx

```diff
@@ -73,7 +74,22 @@ export default function BookingFlowCalendar() {
 						</div>
 					))}
 				</div>
-				{calendarWithAvailability &&
+				{availabilityLoading && (
+					<div>
+						{Array.from({ length: 5 }).map((_, i) => (
+							<div key={i} className={styles.calendarRow}>
+								{Array.from({ length: 7 }).map((_, j) => (
+									<button
+										key={j}
+										disabled={true}
+										className={styles.calendarDay}
+									></button>
```

Comment:
@dawidurbanski I would probably use <div> instead of a <button> here. IMHO there's no need to provide any semantic value here, but this is nit-picking ;).